src/org/labkey/test/components/ui/OmniBox.java

```diff
@@ -140,32 +119,34 @@ private boolean isOpen()
 
     private OmniBox close()
     {
-        elementCache().input.sendKeys(Keys.ESCAPE);
-        getWrapper().waitFor(()-> !isOpen(), 500);
+        if (isOpen())
+        {
+            elementCache().input.sendKeys(Keys.ESCAPE);
+            WebDriverWrapper.waitFor(() -> !isOpen(), "Didn't close omnibox popup", 500);
+        }
         return this;
     }
 
     public boolean isEditing()  // there is always an input; 'editing' means it currently has a value
     {
-        return !Locators.editingValue.findElement(this)
-                .getAttribute("value").isEmpty();
+        return !elementCache().input.getAttribute("value").isEmpty();
     }
 
-    public OmniBox stopEditing()    // click just outside the omnibox, above/center
+    public OmniBox stopEditing()
```

Comment:
I know this is a legacy method, but it just feels a little strange to have as a public method. 
Wouldn't a better design be to have an appendFilter or addFilter method that would add to the existing filter and that method takes care of editing "life cycle" of the OmniBox?